run.sh

```diff
@@ -7,14 +7,39 @@ set -e
 mkdir /tmp/build
 cd /tmp/build
 
+BUILD_FILE="private-build-plans.toml"
+BUILD_PARAM="contents::iosevka"
+
+CUSTOM_BUILD_FILE=false
+
+# Check the input
+if [[ -f "/build/$BUILD_FILE" ]]; then
+    echo "Found custom build-plans file: $BUILD_FILE"
+    CUSTOM_BUILD_FILE=true
+    if [[ -z $1 ]]; then
+        # Get the name of the first build plan when the user does not provide
+        # custom build arguments (automatic mode)
+        PLAN_NAME=$(grep -Po -m 1 '(?<=buildPlans.)[^\]]*' /build/$BUILD_FILE)
+        BUILD_PARAM="contents::$PLAN_NAME"
+    else
+        # User knows what they are doing and provided custom build arguments
+        # (manual mode)
+        BUILD_PARAM="$1"
```

Comment:
@avivace `$1` should change to `$@` to support several fonts build with in one pass.